ghost/README.md

```diff
@@ -149,7 +149,7 @@ version: '3.1'
 services:
 
   ghost:
-    image: ghost:4-alpine
+    image: ghost:5
```

Comment:
I'm not opposed to adding these updates to the docker-compose yaml file, but they do need to be in [`ghost/stack.yml`](https://github.com/docker-library/docs/blob/d9a9f142121491b0c43968548bb76eba9e6b15b7/ghost/stack.yml) since the `ghost/README.md` file is generated.

Technically, the compose file will work as-is on docker-compose while still preserving the image-defined volumes when creating new containers (thus preserving state), they just won't have "pretty" names to keep track of them.